packages/js/src/ui/helpers/formatToRelativeTime.ts

```diff
@@ -0,0 +1,39 @@
+const SECOND = 1000;
+const MINUTE = SECOND * 60;
+const HOUR = MINUTE * 60;
+const DAY = HOUR * 24;
+const YEAR = DAY * 365;
+
+const UNITS = {
+  year: YEAR,
+  month: YEAR / 12,
+  day: DAY,
+  hour: HOUR,
+  minute: MINUTE,
+  second: SECOND,
+};
+
+const DEFAULT_LOCALE = 'en';
+
+export function formatToRelativeTime({
```

Comment:
💭 **thought (non-blocking):** We need to think about the text length as it looks heavy and takes a lot of space IMO, maybe we can generate a few examples and share them with Nikolay to get feedback.